plugins/action/system/iptables_state.py

```diff
@@ -45,13 +45,21 @@ def _async_result(self, module_args, task_vars, timeout):
         Retrieve results of the asynchonous task, and display them in place of
         the async wrapper results (those with the ansible_job_id key).
         '''
+        async_status = self._task.copy()
+        async_status.args = module_args
+        async_status.action = 'async_status'
+        async_status.async_val = 0
+        async_action = self._shared_loader_obj.action_loader.get(
+            'ansible.legacy.async_status', task=async_status, connection=self._connection,
+            play_context=self._play_context, loader=self._loader, templar=self._templar,
+            shared_loader_obj=self._shared_loader_obj)
```

Comment:
Not sure whether loading it once for several invocations ~~with different parameters~~ is correct. @bcoca do you know?